en/includes/guides/authentication/social-login/add-microsoft-login.md

```diff
@@ -161,7 +161,7 @@ Follow the steps given below.
 !!! note
     When a user successfully logs in with Microsoft for the first time, a **user** account is created in the {{ product_name }} Console with the Microsoft username. Microsoft will manage this new user account.
 
-## Add groups to the connection
+## Map groups with {{product_name}}
```

Comment:
We are not mapping any IDP group to IS/Asgardeo's entity here.
Here we define the possible group values which come from the external connection during user authentications